marlowe-playground-client/src/HaskellEditor.purs

```diff
@@ -62,7 +68,9 @@ handleAction settings Compile = do
     Nothing -> pure unit
     Just contents -> do
       assign _compilationResult Loading
-      result <- RemoteData.fromEither <$> (runExceptT $ flip runReaderT settings $ (Server.postContractHaskell $ SourceCode contents))
+      let
+        baseURL = view (_params <<< _Newtype <<< to _.baseURL) settings
```

Comment:
Am I going mad? I can't see where `baseURL` is used...

Reply:
good point, it's not needed